### ISSUE TITLE
Support for ng-class,ng-style,ng-bind-html not present

### ISSUE THREAD

**Issue by pratikp91** [allenhwkim/angular-template#10]
Thanks for this library. But could you provide the above directives support

**Comment by lf94** [comment]
I suggest just using regular templates.

```
<div class="{{ functionsThatOutputsCssClasses }}" style="{{ functionThatOutputsStyle }}"></div>
```

I remember I was able to output HTML too. I don't think angular-template cares about safe/unsafe HTML.

Keep in mind this is a VERY simple implementation of Angular 1.x.

**Comment by lf94** [comment]
I would support these directives but only if I had some incentive...if you know what I mean, haha.

**Comment by FDIM** [comment]
I'll see what I can do. As it is now, ng-bind-html is not needed at all, you can just put your html in a variable and use it in a normal expression.

**Comment by FDIM** [comment]
I've introduced ht-class / ng-class support and restructured internals - now it's more modular and should be easy to extend. @pratikp91 are you up for a small challenge? :)

P.S. Package is not published to npm yet

**Comment by allenhwkim** [comment]
@FDIM thanks, you are the man!

**Comment by FDIM** [comment]
I've just published v2.1.0 that includes support for ng-class, ng-style, ng-bind, ng-bind-html. The last 2 doesn't bring much value, but are still nice for compatibility.
